adafruit_gc_iot_core.py

```diff
@@ -354,7 +354,7 @@ class Cloud_Core:
     """CircuitPython Google Cloud IoT Core module.
 
     :param ESP_SPIcontrol esp: ESP32SPI object.
-    :param dict secrets: Secrets.py file.
+    :param dict secrets: dictonary of settings.
```

Comment:
```suggestion
    :param dict secrets: dictionary of settings.
```
Should this be called `secrets` or `settings`? If the latter, then the arg name will need to change.

Reply:
Clearly you have learned I'm a horrible speller...